Stripe/STPPaymentContext.m

```diff
@@ -484,7 +471,7 @@ - (void)requestPayment {
         if (!self.selectedPaymentMethod) {
             [self presentPaymentMethodsViewControllerWithNewState:STPPaymentContextStateRequestingPayment];
         }
-        else if (self.configuration.requiredShippingAddressFields != STPBillingAddressFieldsNone &&
```

Comment:
yeah, i guess you can compare arbitrary enums if they're the same underlying type :(